pipeline.jsonc

```diff
@@ -51,7 +51,7 @@
         "test":
         [
             {
-                "commit": "5160fa853650669d245e91dccdd5b0275b62748d"
+                "commit": "0090b4bcdbf20db925d290f7a2122ed84c6caf87"
```

Comment:
Do you know why this needed to change?

Reply:
My previous change of this setting after switching the default backing store from ROCKSDB to CHAINBASE was overlapped by a recent merge of another PR, which apparently unrolled this setting back to an earlier commit.